newIDE/app/src/EventsSheet/index.js

```diff
@@ -562,10 +566,31 @@ export default class EventsSheet extends React.Component<Props, State> {
         ? parameterContext.domEvent.currentTarget
         : null,
       inlineEditingChangesMade: false,
+      inlineEditingPreviousValue: instruction.getParameter(parameterIndex),
     });
   };
 
-  closeParameterEditor = () => {
+  closeParameterEditor = (
+    event: SyntheticKeyboardEvent<HTMLInputElement>,
```

Comment:
Don't pass event if you don't need it, replace it by a boolean with a name that describes what it does.

Reply:
For example, this can be `shouldCancel`, which is true/false => much clearer :)